moveit_core/robot_state/src/attached_body.cpp

```diff
@@ -80,6 +80,19 @@ void AttachedBody::setScale(double scale)
   }
 }
 
+void AttachedBody::computeTransform(const Eigen::Isometry3d& parent_link_global_transform)
+{
+  // update collision body transforms
+  for (std::size_t i = 0; i < global_collision_body_transforms_.size(); ++i)
+    global_collision_body_transforms_[i] = parent_link_global_transform * attach_trans_[i];
```

Comment:
Can we make `attach_trans_` more verbose? `attached_shape_transforms_` or so? I find its description from 8 years ago not super helpful, either: 

> The constant transforms applied to the link (needs to be specified by user)

How about "The transforms from the link to each geometry of the attached body"?

Sorry if this inflates the scope too much.